### ISSUE TITLE
Object.dirname is deprecated.

### ISSUE THREAD

**Issue by CalgaryMichael** [elarivie/linter-mypy#13]
I have recently gotten the following error after updating Atom.
Relevant information:
* Atom: 1.23.3
* OS: macOS 10.13.2

Argument to `path.dirname` must be a string
```
Object.dirname (/Applications/Atom.app/Contents/Resources/app/src/electron-shims.js:9:10)
Object.lintPath (/Users/calgarymichael/.atom/packages/linter-mypy/lib/init.coffee:531:23)
Object.lint (/Users/calgarymichael/.atom/packages/linter-mypy/lib/init.coffee:846:28)
a.factory (/Users/calgarymichael/.atom/packages/atom-ide-ui/modules/atom-ide-ui/pkg/atom-ide-diagnostics/lib/services/LinterAdapter.js:252:42)
a._callFactory (/Users/calgarymichael/.atom/packages/atom-ide-ui/node_modules/rxjs/bundles/Rx.min.js:107:23)
a.tryDefer (/Users/calgarymichael/.atom/packages/atom-ide-ui/node_modules/rxjs/bundles/Rx.min.js:106:442)
```

**Comment by elarivie** [comment]
Thanks @CalgaryMichael for the bug report, the fix is now available in linter-mypy v2.5.2

**Comment by CalgaryMichael** [comment]
No problem. Thank you for staying on top of this repositories issues!
